tox.ini

```diff
@@ -7,8 +7,6 @@ deps =
   flake8
   {toxinidir}/py
   {toxinidir}/pytest
```

Comment:
According to @jgraham we should be running the wpt-tools with upstream py.test rather than our vendored copy, so this needs to just be `pytest` (and we should like `py` just automatically get pulled in as a dependency rather than explicitly listing it).